orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -33,6 +34,12 @@ def _ortvalue_to_dlpack(ortvalue):
 
 def _ortvalue_from_dlpack(dlpack_tensor):
     return OrtValue(C.OrtValue.from_dlpack(dlpack_tensor))
+class Verbosity(IntEnum):
```

Comment:
Nit: Add 2 blank lines before class definition (looks like a merge artefact).

Reply:
Running autopep8 in the whole file